Rakefile

```diff
@@ -223,3 +223,40 @@ namespace :crowbar do
     end
   end
 end
+
+namespace :test do
+  task dependencies: [] do
+    if ENV["TRAVIS"]
+      system("pip install bashate")
+    else
+      system("sudo pip install bashate")
+    end
+    exit $?.exitstatus
+  end
+
+  desc "Run bashate tests"
+  task bashate: [:dependencies] do
+    script = "scripts/install-chef-suse.sh"
```

Comment:
But i can't loop over 1 file, usually a loop is used for more items, like the elements of an array.